plugins/services/src/js/components/forms/ContainerServiceFormSection.js

```diff
@@ -352,7 +357,9 @@ class ContainerServiceFormSection extends Component {
               Memory (MiB)
             </FieldLabel>
             <FieldInput
+              min="0.001"
```

Comment:
Hm... The API spec actually specifies `0.001` as minimum, thus I'd say we should keep it as it is. 

https://github.com/mesosphere/marathon/blob/master/docs/docs/rest-api/public/api/v2/types/resources.raml#L18